geoportailv3/static/js/profile/profiledirective.js

```diff
@@ -181,6 +206,27 @@ app.ProfileController = function($scope) {
 
 
   this['point'] = null;
+
+  $scope.$watch(goog.bind(function() {
+    return this['profileData'];
+  }, this), goog.bind(function(newVal, oldVal) {
+    if (goog.isDef(newVal)) {
+      var i;
+      var len = newVal.length;
+      var lineString = new ol.geom.LineString([],
+          /** @type {ol.geom.GeometryLayout} */ ('XYM'));
```

Comment:
Use `ol.geom.GeometryLayout.XYM` instead of type-casting. (Do not forget to add `goog.requires('ol.geom.GeometryLayouyt')`.)